roles/manage_dbpatches/tasks/manage_dbpatches.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Set the os variable
+  set_fact:
+    os: "{{ ansible_distribution }}{{ ansible_distribution_major_version }}"
+
+- name: Check support for Operating System
+  fail:
+    msg: "Operating System = {{ os }} not supported."
+  when: os not in supported_os
+
+- name: Check supported versions for Database engine
+  fail:
+    msg: "Database Engine Version = {{ pg_version }} not supported.
+          Supported versions are {{ supported_pg_version }}"
+  when: pg_version|int not in supported_pg_version
+
+- name: Check support for efm
+  fail:
+    msg: "efm version = {{ efm_version }} not supported.
+         Supported versions are {{ supported_efm_version }}"
+  when: efm_version | string not in supported_efm_version
+
+- name: Reference variables
+  include_vars: "{{ pg_type }}_{{ ansible_os_family }}.yml"
+
+- name: Gather services facts
+  service_facts:
+
+- name: Gather the EFM cluster informations
+  set_fact:
+    efm_cluster_nodes: "{{ lookup('edb_devops.edb_postgres.efm_nodes', wantlist=True) }}"
+  run_once: true
+
+- name: Get the primary information
+  set_fact:
+    efm_inventory_primary_public_ip: "{{ node.ansible_host }}"
+    efm_inventory_primary_private_ip: "{{ node.private_ip }}"
+    efm_inventory_primary_hostname: "{{ node.inventory_hostname }}"
+  when: node.node_type == 'primary'
+  loop: "{{ efm_cluster_nodes }}"
+  loop_control:
+    loop_var: node
+  run_once: true
+  no_log: "{{ disable_logging }}"
+
+- name: Validate EFM cluster with inventory state
+  ansible.builtin.import_tasks: validate_efm_cluster.yml
+
+- name: Update standby servers to latest package one at a time
+  throttle: 1
+  block:
+  - ansible.builtin.include_tasks: apply_db_patches.yml
```

Comment:
Please add a name for this task (inside the `block` section). That's the reason why ansible-lint is not happy.

Reply:
Fixed.